D98 ADVECV instrumentation (MOMEN2ND.f: entry/exit hooks). Generated by diff -u against the PRISTINE MOMEN2ND.f (2026-10-05).

--- MOMEN2ND.f.orig
+++ MOMEN2ND.f
@@ -88,6 +88,7 @@
      &               HAVE_SOUTH_POLE = HAVE_SOUTH_POLE,
      &               HAVE_NORTH_POLE = HAVE_NORTH_POLE)
 
+      call ffdb_advecv_in(IM,JM,LM,DT1,U,V,MMEAN,MBEFOR,UT,VT,MAFTER)
       IF(MODD5K.LT.MRCH) CALL DIAG5F (U,V)
 
       DT2=DT1/2.
@@ -494,6 +495,7 @@
       DUT(:,J_0S:J_1,:) = 0
       DVT(:,J_0S:J_1,:) = 0
 C
+      call ffdb_advecv_out(IM,JM,LM,UT,VT)
       RETURN
       END SUBROUTINE ADVECV
 
